ui/settings.py

```diff
@@ -216,6 +217,27 @@
 SECURE_HSTS_SECONDS = int(os.getenv('SECURE_HSTS_SECONDS', '16070400'))
 SECURE_HSTS_INCLUDE_SUBDOMAINS = True
 
+# HEADER/FOOTER URLS
+# EXPORTING PERSONAS
+EXPORTING_NEW = os.environ["EXPORTING_NEW"]
```

Comment:
We should default to prod links on all these

Reply:
the header/footer context processor does that if the link isn't specified. they're listed in directory-constants https://github.com/uktrade/directory-constants/blob/master/directory_constants/constants/urls.py

Reply:
okay this was also breaking the tests. have fixed now